src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import './DropdownMenu.scss';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[];
+  selectedPerson: Person | null;
+  onSelect: (person: Person) => void;
+};
+
+export const DropdownMenu: FC<Props> = ({
+  people,
+  selectedPerson,
+  onSelect,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map(person => {
+          const { name, sex } = person;
+
+          return (
+            <div
+              key={person.slug}
+              className={cn('dropdown-item', {
+                'has-background-grey-lighter': selectedPerson === person,
+              })}
+              data-cy="suggestion-item"
+              onClick={() => onSelect(person)}
```

Comment:
The naming convention for event handlers in React suggests that the name should start with 'handle' and end with the type of event you are handling. In this case, a better name would be 'handleSelectPerson'. You can read more about the naming conventions [here](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364).